packages/docs/docs/careplans/tasks/tasks.md

```diff
@@ -0,0 +1,140 @@
+| Assistant                  | 449161006 ([Physician assistant](https://browser.ihtsdotools.org/?perspective=full&conceptId1=449161006&edition=MAIN/2023-03-31&release=&languages=en))        | 29-1071 (Physician Assistants)                            |
+
+Below is an example of a `Task.performerType` [CodeableConcept](/docs/fhir-basics#codeable-concepts-standarding-data) using both SNOMED and SOC systems.
+
+```ts
+{
+  resourceType: 'Task',
+  // ...
+  performerType: [
+    {
+      text:'Medical Billing Specialist',
+      coding:[
+        // Snomed
+        {
+          code:'1251542004',
+          system: 'http://snomed.info/sct',
+          display: 'Medical Coder'
+        },
+        // US SOC
+        {
+          code:"29-2072"
+          system: "https://www.bls.gov/soc"
+        }
+      ],
+    }
+  ]
+}
+```
+
+## Task focus
+
+The `Task.focus` element tracks the FHIR resource being _operated on_ by this task, known as the "focal resource". See the [Examples](#examples) section below for examples of focal resources in common scenarios.
+
+Well maintained `Task.focus` elements are critical data hygiene that streamlines operations and analytics . Making sure that every [`Task`](/docs/api/fhir/resources/task) has a populated`focus` reference will make it easier to find all touch points for a given clinical resource, spot operational bottlenecks and calculate turn-around-times, conversions, and care quality metrics as your implementation scales.
+
+## Task start / due dates
+
+The `Task.restriction.period` field describes the time period over which the [`Task`](/docs/api/fhir/resources/task) should be fulfilled, with `Task.restriction.period.end` representing the _due date_, and `Task.restriction.period.start` representing the (potentially optional) start date.
+
+## Task comments
+
+`Task.note`` can be used to capture narrative text that is not represented elsewhere in the resource.
+
+The most common usage for this field is to record comments from the task assignee as they work on the task. When used this way, it is a best practice to include the `author` and `time` fields in the [`Annotation`](/docs/api/fhir/datatypes/annotation).
+
+## Subtasks
+
+`Tasks` can be organized into a hierarchical structure to create subtasks, sub-tasks, etc. To represent this hierarchy, subtasks should reference their parent using the using the `Task.partOf` element. `Task.partOf` is a searchable field, which can be used to query all sub-tasks of a given task, and can be combined with the [`_revinclude`](/docs/search/includes#_include-and-_revinclude) and [`:iterate`](/docs/search/includes#iterate-modifier) directives to query the entire [`Task`](/docs/api/fhir/resources/task) tree.
+
+:::caution
+
+While this functionality is powerful, it can be complex to maintain and operationalize. Medplum recommends that most implementations start with a single-level [`Task`](/docs/api/fhir/resources/task) hierarchy, and gradually add depth over time.
```

Comment:
instead of this, phrase as "task hierarchy" or similar descriptive phrase

Reply:
Not sure I understand. Currently reads as 
> Medplum recommends that most implementations start with a single-level `Task`hierarchy

Reply:
"While task hierarchy functionality is powerful, it can be complex to maintain and operationalize. Medplum recommends that most implementations start with a single-level [`Task`](/docs/api/fhir/resources/task) hierarchy, and gradually add depth over time."